docs/dunst.pod

```diff
@@ -264,6 +264,15 @@ The progress value can be set with a hint, too.
 
 =back
 
+=head1 ACTIONS
+
+Dunst allows interaction with actions in notifications. If "show_indicators" is
+true, a "(A)" will be appended to the notification title. The "context"
+keybinding can be used to interact with these actions (regardless of the
+indicators being displayed).
+
+This feature requires "dmenu" to be installed.
```

Comment:
Should also mention that it doesn't *require* dmenu and it can also work with dmenu drop-in replacements if the `dmenu` setting is set to use them.